apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -49,22 +81,426 @@ export const getPageInfo: GetPageNavInfo = ({
+              defaultMessage: "We successfully received your application",
+              id: "k0//XC",
+            }),
+          );
+          navigate(nextStep);
+        }
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: Error: submitting application failed",
+            id: "Uysn1m",
+          }),
+        );
+      });
+  };
+
+  const cancelPath = applicantDashboard ? paths.dashboard() : paths.myProfile();
+  const editPaths = {
+    resume: paths.applicationResume(application.id),
+    education: paths.applicationEducation(application.id),
+    skills: paths.applicationSkills(application.id),
+    screeningQuestions: paths.applicationQuestions(application.id),
+  };
+
+  const nonEmptyExperiences = experiences?.filter(notEmpty) ?? [];
+  const hasSomeExperience = !!experiences.length;
+  const educationRequirementExperiences =
+    application.educationRequirementExperiences
+      ? application.educationRequirementExperiences.filter(notEmpty)
+      : [];
 
-  return <Heading data-h2-margin-top="base(0)">{pageInfo.title}</Heading>;
+  const categorizedEssentialSkills = categorizeSkill(
+    application.poolAdvertisement?.essentialSkills,
+  );
+
+  const screeningQuestions =
+    application.poolAdvertisement?.screeningQuestions?.filter(notEmpty) || [];
+  const screeningQuestionResponses =
+    application.screeningQuestionResponses?.filter(notEmpty) || [];
+  return (
+    <section>
+      <Heading data-h2-margin-top="base(0)">{pageInfo.title}</Heading>
+      <div data-h2-margin-bottom="base(x2)">
+        <p data-h2-margin-bottom="base(x1)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Before continuing, it’s important that you understand the following notes about your application:",
+            id: "hX7Z1l",
+          })}
+        </p>
+        <ol>
+          <li>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "When you submit your application, a copy of your profile will be created as a part of your application.",
+                id: "WWoPd/",
+              })}
+            </p>
+          </li>
+          <li>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "Changes made to your profile after submitting will not be updated on your application.",
+                id: "RvafWw",
+              })}
+            </p>
+          </li>
+          <li>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "You are still encouraged to keep your profile up to date, as updated versions will be used at later steps of the hiring process.",
+                id: "TIU07I",
+              })}
+            </p>
+          </li>
+        </ol>
```

Comment:
I beleive this should be an unordered list (`ul`) since the order doesn't matter and [the list is not sequential](https://www.w3.org/WAI/tutorials/page-structure/content/#ordered-list:~:text=Ordered%20lists%20are%20used%20for%20sequential%20information%20and%20are%20automatically%20enumerated%20by%20the%20browser.).

Reply:
Fixed by [c69a436](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/c69a436656a8d6a449d0bf4f065972b6d95382f4)